nncf/torch/quantization/quantize.py

```diff
@@ -213,3 +218,45 @@ def send_to_device(tensor):
     compressed_model.disable_dynamic_graph_building()
 
     return compressed_model
+
+
+def quantize_impl_experimental(
+        model: torch.nn.Module,
+        calibration_dataset: Dataset,
+        preset: QuantizationPreset,
+        target_device: TargetDevice,
+        subset_size: int,
+        fast_bias_correction: bool,
+        model_type: Optional[ModelType] = None,
+        ignored_scope: Optional[IgnoredScope] = None) -> torch.nn.Module:
+    """
+    Experimental implementation of the `quantize()` method for the PyTorch backend.
+    """
+    if fast_bias_correction is False:
```

Comment:
Neither is the `Fast-` nor that just `BiasCorrection` supported for now for the PyTorch PTQ, isn't it?

Reply:
Yes, I've added warning to make it clearer

Reply:
Let's add it for `quantize_impl_experimental` as well.

Reply:
Sure, done!